json-spec/extensions/stac-collection-spec.md

```diff
@@ -0,0 +1,154 @@
+# STAC Collection Extension Spec
+
+A group of STAC `Item` objects from a single source usually shared common metadata. This is especially true with satellite imagery that uses the STAC EO Extension. Rather than including these common metadata fields on every `Item`, they can be grouped into a `Collection`.
+
+## Collection Extension Description
+
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| collection_name | string | Collection Name | A name given to the Collection
+| collection_description | string (optional) | Collection Description | A human readable description of the collection
+
+A `Collection` does not have many specific fields, as it may contain any fields that are in the core spec as well as any other extension. If a field is specified in `Collection`, it will be ignored in any `Item` that links to that `Collection`. This is important because a `Collection` is the metadta that is common across all `Item` objects. If a field is variable at all, it should be part of the `Item` record.
+
+#### Linking to a `Collection`
```

Comment:
I'm sorta wondering if we should call it 'collection metadata' or 'collection info' or something. Especially once you move from earth observation world to WFS world - the 'collection' is the set of data, it's not the common metadata. And the logical endpoint in WFS is something like /collections/<collection_name>/(?) Like in that path something like /collections/buildings/info.json would make more sense to me than like /collections/buildings/collection.json

I'm also still contemplating how this could cover the 'root catalog' stuff as well - https://github.com/radiantearth/stac-spec/tree/dev/static-catalog#catalog Like the fields in it - contact, formats, keywords, homepage, etc could make sense in the same file? Maybe it's that the static catalog link would be required to implement both the collection schema and the 'catalog' schema, and the catalog schema is just that it has links?

Reply:
I went with Collection for two reasons. 

- It seemed compatible with WFS, although I haven't thought in detail about how it would work in practice, so your points are well taken.
- For NASA data, this is what a "Collection" is - a group of similar data, some might refer to it as a product.

I haven't thought about the root catalog stuff at all.

Reply:
Cool. I'll see if I can come up with something tomorrow (thursday).